code/game/machinery/computer/buildandrepair.dm

```diff
@@ -83,6 +205,8 @@
 			return ITEM_INTERACT_BLOCKING
 
 /obj/structure/frame/computer/crowbar_act(mob/living/user, obj/item/tool)
+	. = ITEM_INTERACT_BLOCKING
```

Comment:
I explicitly avoided doing this, to let people bash frames with crowbars while in combat mode (like jaws of life or something) 

...But I realize I only did it for two states. (`FRAME_COMPUTER_STATE_BOARD_SECURED` and `FRAME_COMPUTER_STATE_WIRED`)